### ISSUE TITLE
Synchronising release versions with docker image versions

### ISSUE THREAD

**Issue by mgrbyte** [WormBase/wormbase_rest#17]
Currently, the latest tag in git is `0.0.2`, but the image in our docker repository (ECR) has the tag 0.0.1.

After merging the "release branch", I think we need to run the following (in order):

``` bash
git tag ${VERSION}
git checkout ${VERSION}
make docker-tag
make docker-push-ecr
```

We could wrap these commands up in  a `release` part to the `Makefile`/script that does the required steps.

**Comment by adamjohnwright** [comment]
I have created a release (0.0.3) with the changes in the merge request. I have also run the commands necessary to get a container build with the tag into ECR. The only thing to note is that I created the tag with gitflow.

At this point I am not sure if a script will be necessary. I am neither for or against it. I think we should go through a couple release cycles before we determine it to be necessary. The main reason for this is because there are already very few commands necessary to make a release.
